packages/config/index.js

```diff
@@ -129,17 +133,28 @@ BemConfig.prototype.level = function(pathToLevel) {
  * @returns {Promise}
  */
 BemConfig.prototype.library = function(libName) {
-    return this.get().then(function(config) {
-        var libs = config.libs;
+    return this.get()
+        .then(function(config) {
+            var libs = config.libs,
+                lib = libs && libs[libName];
 
-        if (!libs) { return; }
+            if (lib !== undefined && typeof lib !== 'object') {
+                return Promise.reject('Invalid `libs` format');
+            }
 
-        var lib = libs[libName];
+            var cwd = lib && lib.path || path.resolve('node_modules', libName);
```

Comment:
the only one property of library is `path` and it is optional? Config with `libs: { 'lego-on-react': {} }` seems weird for me)